schemas/dto/jobLogEntryDto.json

```
@@ -53,8 +53,8 @@
       "description": "Status of action performed above authority",
       "$ref": "./actionStatus.json"
     },
-    "orderActionStatus": {
-      "description": "Status of action performed above order, value MULTIPLE if multiple orders were affected",
+    "poLineActionStatus": {
+      "description": "Status of action performed above poLine, value MULTIPLE if multiple poLine were affected",
```

```suggestion
      "description": "Status of action performed above poLine",
```
we do not expect cases where multiple status would be relevant for poLines